harvard-lts/CURIOSity
# Add Numbers to Search Results

**Add Numbers to Search Results**
* * *

**JIRA Ticket**: [LTSMAINT-759](https://jira.huit.harvard.edu/browse/LTSMAINT-759)

# What does this Pull Request do?
Add numbers back to search results list.

# How should this be tested?
* Spin up Curiosity locally off of this branch
* Navigate to the search results page of any exhibit
* Numbers should appear to the left of item titles in the list view only (not in gallery or masonry) and match the font size and weights of the titles
* Numbers will also appear in "No thumbnail or viewer" theme (Exhibit Dashboard --> Appearance --> No thumbnail or viewer)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@phil-plencner-hl @dl-maura @enriquediaz
